tikv/kv.go

```diff
@@ -285,7 +285,7 @@ func NewKVStore(uuid string, pdClient pd.Client, spkv SafePointKV, tikvclient Cl
 		clusterID:       pdClient.GetClusterID(context.TODO()),
 		uuid:            uuid,
 		oracle:          o,
-		pdClient:        pdClient,
+		pdClient:        pdClient.WithCallerComponent("kv-store"),
```

Comment:
How about defining and describing one enum not using string?

Reply:
How about defining them in pd repo? If you agree, I can open a pr to do it.